ui/packages/catalog-ui-search/src/main/webapp/component/result-form/result-form.view.js

```diff
@@ -0,0 +1,180 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+/* global setTimeout */
+const Marionette = require('marionette')
+const $ = require('jquery')
+const template = require('./result-form.hbs')
+const CustomElements = require('js/CustomElements')
+const store = require('js/store')
+const PropertyView = require('component/property/property.view')
+const Property = require('component/property/property')
+const metacardDefinitions = require('component/singletons/metacard-definitions')
+const Loading = require('component/loading-companion/loading-companion.view')
+const _ = require('underscore')
+const announcement = require('component/announcement')
+const ResultFormCollection = require('component/result-form/result-form')
+const Common = require('js/Common')
+const ResultForm = require('component/search-form/search-form')
+
+module.exports = Marionette.LayoutView.extend({
+  template: template,
+  tagName: CustomElements.register('result-form'),
+  modelEvents: {},
+  events: {
+    'click .editor-edit': 'edit',
+    'click .editor-cancel': 'cancel',
+    'click .editor-save': 'save'
+  },
+  regions: {
+    basicTitle: '.basic-text',
+    basicDescription: '.basic-description',
+    basicAttribute: '.basic-type',
+    basicAttributeSpecific: '.basic-type-specific'
+  },
+  ui: {},
+  filter: undefined,
+  onBeforeShow: function () {
+    this.model = this.model._cloneOf ? store.getQueryById(this.model._cloneOf) : this.model
+    this.setupTitleInput()
+    this.setupDescription()
+    this.setupAttributeSpecific()
+    this.turnOnLimitedWidth()
+    this.edit()
+  },
+  setupAttributeSpecific: function () {
+    let currentValue = this.model.get('descriptors') !== '{}' || this.model.get('descriptors') !== '[]' ? this.model.get('descriptors') : []
```

Comment:
❓ Do you want to be comparing to `{}` and `[]` instead of the literal strings `'{}'` and `'[]'`?